auth/jwt/README.md

```diff
@@ -0,0 +1,123 @@
+# package auth/jwt
+
+`package auth/jwt` provides a set of interfaces for service authorization through [JSON Web Tokens](https://jwt.io/).
+
+## Usage
+
+NewParser takes a key function and an expected signing method and returns an `endpoint.Middleware`. 
+The middleware will parse a token passed into the context via the `jwt.JWTTokenContextKey`. 
+If the token is valid, any claims will be added to the context via the `jwt.JWTClaimsContextKey`.
+
+```go
+import (
+	stdjwt "github.com/dgrijalva/jwt-go"
+    
+    "github.com/go-kit/kit/auth/jwt"
+    "github.com/go-kit/kit/endpoint"
+)
+
+func main() {
+	var exampleEndpoint endpoint.Endpoint
+	{
+		keyFunc := func(token *stdjwt.Token) (interface{}, error) { return []byte("SigningString"), nil }
+		jwtParser := jwt.NewParser(keyFunc, stdjwt.SigningMethodHS256)
```

Comment:
Hmm, I could add a jwt.NewDefaultParser() function. I also don't know off hand what the desired default would be. https://jwt.io defaults to `HS256` (`SHA-256`) but I could see `HS512` (`SHA-512`) being pretty common as well.